src/ui/src/builder/settings/BuilderSettingsProperties.vue

```diff
@@ -244,7 +247,11 @@ import BuilderFieldsWriterResourceId from "./BuilderFieldsWriterResourceId.vue";
 import BuilderFieldsComponentId from "./BuilderFieldsComponentId.vue";
 import BuilderFieldsComponentEventType from "./BuilderFieldsComponentEventType.vue";
 import { useFieldsErrors } from "@/renderer/useFieldsErrors";
-import WdsTitle2 from "@/wds/WdsTitle2.vue";
+import WdsTabs, { type WdsTabOptions } from "@/wds/WdsTabs.vue";
+
+defineProps({
+	isReadOnly: { type: Boolean, required: true },
```

Comment:
I think for this kind of props, you can use `required: false`

```suggestion
	isReadOnly: { type: Boolean },
```

Reply:
I prefer to keep required: true for isReadOnly, since it is a critical behavioral prop and part of the component's contract